.env

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=root

```

Comment:
Rename to .env.sample and don't specify variables like 
`MYSQLDB_USER=/your db user/`